sample-apps/python/django_frontend_service/frontend_service_app/views.py

```diff
@@ -63,6 +103,11 @@ def aws_sdk_call(request):
     agent_based_histogram.record(random.randint(100, 1000), {"Operation": "histogram"})
     agent_based_gauge.add(random.randint(-10, 10), {"Operation": "gauge"})
 
+    if custom_pipeline_counter:
```

Comment:
Not blocking: Slightly cleaner
```suggestion
    if pipeline_meter:
```